File: Entities/DevolucionPorPaquete.cs
namespace Entidades
{
    using System;

    public class DevolucionPorPaquete
    {
        private int varIdDevolucion;
        private int varIdPaquete;

        public int IdDevolucion
        {
            get => 
                this.varIdDevolucion;
            set => 
                this.varIdDevolucion = value;
        }

        public int IdPaquete
        {
            get => 
                this.varIdPaquete;
            set => 
                this.varIdPaquete = value;
        }
    }
}

File: Entidades/DevolucionPorPaquete.vb
Public Class DevolucionPorPaquete

    Private varIdDevolucion As Integer
    Public Property IdDevolucion() As Integer
        Get
            Return varIdDevolucion
        End Get
        Set(ByVal value As Integer)
            varIdDevolucion = value
        End Set
    End Property


    Private varIdPaquete As Integer
    Public Property IdPaquete() As Integer
        Get
            Return varIdPaquete
        End Get
        Set(ByVal value As Integer)
            varIdPaquete = value
        End Set
    End Property

End Class
